code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp

```diff
@@ -0,0 +1,17 @@
+// Part of cosmos from opengenus foundation
+
+#include <iostream>
+#include <vector>
+#include <algorithm>
+
+int main() {
+    std::vector<int> arr;
+    std::cout << "Keep entering numbers (EOF to stop): ";
+    for (int num; std::cin >> num;) {
+        arr.push_back(num);
+    }
+
+    sort(arr.begin(), arr.end());
```

Comment:
Again, I suggest you actually implement the algorithm.

Reply:
as you also know CPP provides us STL 
for algo contributors have added the same program in C